assets/helpers/config.js

```diff
@@ -0,0 +1,18 @@
+// @flow
```

Comment:
Why is this file called config? Is it not better to call it user since only cares about users data.